all_test.go

```diff
@@ -198,31 +210,33 @@ func TestRenameIdentity(t *testing.T) {
 	}
 
 	if _, err := c.GetTokenFromSession(session); err != ErrInvalidSessionToken {
-		t.Fatalf("All sessions for 'joe' should have been invalidated by rename")
+		t.Fatalf("All sessions for 'joe' should have been invalidated by rename, %s", err)
 	}
 
 	if _, _, err := c.Login("sarah", "123"); err != nil {
 		t.Fatalf("Login as 'sarah' failed (%v)", err)
 	}
+
+	teardown(c)
 }
 
 func TestResetPassword(t *testing.T) {
 	c := setup1(t)
 
-	if _, err := c.ResetPasswordStart("nobody_ever", time.Now()); err != ErrIdentityAuthNotFound {
+	if _, err := c.ResetPasswordStart(999, time.Now()); err != ErrIdentityAuthNotFound {
```

Comment:
I think it's OK to either sprinkle `999` throughout the code, or always use the constant `notFoundUserId`, but you shouldn't mix the two.